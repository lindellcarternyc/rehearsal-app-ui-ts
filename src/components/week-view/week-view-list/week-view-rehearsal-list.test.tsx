import * as React from 'react'
import * as renderer from 'react-test-renderer'
import WeekViewRehearsalList from './week-view-rehearsal-list'

describe('WeekViewRehearsalList', () => {
  it('renders a list of one rehearsal', () => {
    const times = ['12:00 - 16:00']
    const tree = renderer.create(
      <WeekViewRehearsalList times={times} />
    )
    expect(tree).toMatchSnapshot()
  })

  it('renders a list of multiple rehearsals', () => {
    const times = ['12:00 - 14:30', '14:30 - 16:00']
    const tree = renderer.create(
      <WeekViewRehearsalList times={times} />
    )
    expect(tree).toMatchSnapshot()
  })

  it('renders nothing if there are no rehearsals', () => {
    const tree = renderer.create(
      <WeekViewRehearsalList times={[]} />
    )
    expect(tree).toMatchSnapshot()
  })
})