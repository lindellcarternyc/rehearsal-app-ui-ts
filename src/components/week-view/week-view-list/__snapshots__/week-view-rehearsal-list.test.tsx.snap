// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeekViewRehearsalList renders a list of multiple rehearsals 1`] = `
<div
  className="ui bulleted list"
  role="list"
>
  <div
    className="item"
    data-value={undefined}
    onClick={[Function]}
    role="listitem"
  >
    12:00 - 14:30
  </div>
  <div
    className="item"
    data-value={undefined}
    onClick={[Function]}
    role="listitem"
  >
    14:30 - 16:00
  </div>
</div>
`;

exports[`WeekViewRehearsalList renders a list of one rehearsal 1`] = `
<div
  className="ui bulleted list"
  role="list"
>
  <div
    className="item"
    data-value={undefined}
    onClick={[Function]}
    role="listitem"
  >
    12:00 - 16:00
  </div>
</div>
`;

exports[`WeekViewRehearsalList renders nothing if there are no rehearsals 1`] = `null`;
